crates/driver/openapi.yml

```
@@ -59,6 +60,7 @@ paths:
           $ref: "#/components/responses/InternalServerError"
   /solve:
     post:
+      operationId: solveAuction
```

Here we have `solveAuction`, but in the solver's yaml, just `solve`. We should be consistent across the files.
```suggestion
      operationId: solve
```